core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalWriter.java

```diff
@@ -35,7 +35,7 @@
 @NotThreadSafe
 public class RaftJournalWriter implements JournalWriter {
   private static final Logger LOG = LoggerFactory.getLogger(RaftJournalWriter.class);
-  private static final long FLUSH_TIMEOUT_S = 2;
+  private static final long FLUSH_TIMEOUT_S = 30;
```

Comment:
Should this be configurable?

Reply:
We already have `alluxio.master.journal.flush.timeout.ms`, this timeout is just how long we wait before logging a warning and trying again (if we still haven't exceeded `alluxio.master.journal.flush.timeout.ms`)

Reply:
I'll add a comment